.travis.yml

```diff
@@ -12,6 +12,9 @@ addons:
       - libgles2-mesa-dev
 script:
   - (cd webrender_traits && cargo test --verbose)
+  - (cd webrender_traits && cargo test --verbose --features "ipc")
   - (cd webrender && cargo test --verbose)
+  - (cd webrender && cargo test --verbose --features "ipc")
```

Comment:
`webrender` and `sample` don't have the `ipc` feature, this is only a `webrender_traits` thing, so this line and the one below you added should go away.